tests/json/tilejson/valid/minimal.json

```diff
@@ -0,0 +1,14 @@
+{
+    "tilejson": "3.0.0",
+    "tiles": [
+        "http://localhost:8080/tiles/{zlvl}/{x}/{y}.json"
+    ],
+    "vector_layers": [
```

Comment:
"vector_layers" change to camel case

Reply:
vector_layers without capitalization as in camelCase comes from TileJSON.